src/main/java/com/bobrust/gui/dialog/DrawDialog.java

```diff
@@ -294,10 +310,21 @@ private void showPaletteWarning() {
 		});
 		JOptionPane.showMessageDialog(this, pane, "Could not find the palette", JOptionPane.WARNING_MESSAGE);
 	}
-	
+	public static BufferedImage captureImageFromRect(Rectangle rect) {
+		try {
+			Robot robot = new Robot();
+
+			BufferedImage screenCapture = robot.createScreenCapture(rect);
+
+			return screenCapture;
+		} catch (AWTException e) {
+			e.printStackTrace();
+			return null;
+		}
+	}
 	private boolean findColorPalette() {
 		// Take a screenshot
-		BufferedImage screenshot = RustWindowUtil.captureScreenshot(monitor);
```

Comment:
I'd rather you implement window ui capture screenshot because it's dependant on window scaling. It's complicated when working with multi resolution monitors